index.bs

```diff
@@ -1219,7 +1219,7 @@ following Web IDL.
     dictionary CollectedClientData {
         required DOMString           challenge;
         required DOMString           origin;
-        required DOMString           hashAlg;
+        required DOMString           hashAlgorithm;
```

Comment:
well, AFAICT "alg" is only used in the spec IDL or prose in two places:
```
> grep -iE "alg[;\s}\.]+" index.bs 
        <code>|current|.{{PublicKeyCredentialParameters/alg}}</code> and |op| set to `"generateKey"`. If an error occurs during
        required AlgorithmIdentifier          alg;
```

..(tho my egrep fu is so-so) and so I'm inclined to leave this as-is with the vast majority of the use of [Aa]lgrorithm (85 instances) being the spelled-out form. 

that ok?